### PR TITLE
glimagesink: add modifier support

### PR DESCRIPTION
**Pull request by liuyinhangx**

### CODE REVIEW

**Comment by liuyinhangx** [comment]
@mengker33 @HeJunyan @xhaihao @xuguangxin please help to review, thanks.

**Comment by liuyinhangx** [comment]
@mengker33 @tong1wu @HeJunyan @xhaihao @xuguangxin Would you please help to review the code when you have time? Many thanks.

File: subprojects/gst-plugins-base/gst-libs/gst/video/video-drm-format.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* GStreamer
+ *  Copyright (C) 2022 Intel Corporation
```
Comment (mengker33) [inline subprojects/gst-plugins-base/gst-libs/gst/video/video-drm-format.h:2]: The same space here...

File: subprojects/gst-plugins-base/gst-libs/gst/video/video-drm-format.c [hunk 2]
```diff
@@ -0,0 +1,147 @@
+/* GStreamer
+ *  Copyright (C) 2022 Intel Corporation
```
[the remark this answers is not in the document]
Comment (liuyinhangx) [reply subprojects/gst-plugins-base/gst-libs/gst/video/video-drm-format.c]: Fixed. Thank you, @mengker33

File: subprojects/gst-plugins-base/gst-libs/gst/video/video-drm-format.h (hunk 1 above)
Comment (liuyinhangx) [reply subprojects/gst-plugins-base/gst-libs/gst/video/video-drm-format.h]: Fixed.

**Comment by HeJunyan** [comment]
I did some rework for your first patch about the helper functions, please help to review and rebase onto it,
https://github.com/intel-media-ci/gstreamer/pull/36
thanks

**Comment by liuyinhangx** [comment]
> I did some rework for your first patch about the helper functions, please help to review and rebase onto it, #36 thanks

Ok, I will rebase the code.

**Comment by uartie** [comment]
No activity for a while. Closing to free CI resources.

**Pull request closed without merging**